.gitignore

```diff
@@ -7,3 +7,4 @@ node_modules/
 /dist-assets/
 .eslintcache
 .env
+/.idea
```

Comment:
Remove this please. We don't list artifacts of various IDE. You should do it in your local git config so it applies to all git projects you author.